packages/url_launcher/README.md

```diff
@@ -46,7 +46,7 @@ Common schemes supported by both iOS and Android:
 | Scheme | Example | Action |
 |---|---|---|
 | `http:<URL>` , `https:<URL>` | `http://flutter.io` | Open URL in the default browser |
-| `mailto:<email address>` | `mailto:smith@example.org` | Open <email address> in the default email app |
+| `mailto:<email address>?subject=<subject>&body<body> | `mailto:smith@example.org?subject=News&body=New%20%Flutter%plugin` | Create email to <email address> in the default email app |
```

Comment:
Change 'mailto:smith@example.org?subject=News&body=New%20%Flutter%plugin' to 'mailto:smith@example.org?subject=News&body=New%20Flutter%20plugin'